_automating-workflows/index.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Automating workflows
+nav_order: 1
+has_children: false
+nav_exclude: true
+redirect_from: /automating-workflows/
+---
+
+# Automating workflows
+**Introduced 2.12**
+{: .label .label-purple }
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](Link TBD).    
+{: .warning}
+
+You can automate complex OpenSearch setup and preprocessing tasks by providing templates for common use cases. For example, automating machine learning (ML) setup tasks streamlines the use of OpenSearch ML offerings.
+
+In OpenSearch 2.12, workflow automation is limited to ML tasks.
+{: .info}
+
+OpenSearch use case templates provide a compact description of the setup process in a JSON or YAML document. These templates describe automated workflow configurations for conversational chat or query generation, AI connectors, tools, agents, and other components that prepare OpenSearch as a backend for generative models. 
+
```

Comment:
Can we add a link here to sample templates?  I have PR open https://github.com/opensearch-project/flow-framework/pull/498 which will add some sample templates in this folder.  (I'll have a README there with more detailed instructions shortly.)

https://github.com/opensearch-project/flow-framework/tree/main/sample-templates

Reply:
That's great, thanks! Added.

Reply:
Re: issue on github. Normally, we want the users to follow the progress of the feature, which they can't do by opening an issue. Is there a meta issue or rfc that we can reference?

Reply:
https://github.com/opensearch-project/flow-framework/issues/475 is probably a good place!

Reply:
Thank you! Added. Let me know if there is anything else for this issue. Otherwise, we can merge.

Reply:
I think that's it!